store/splus.js

```diff
@@ -4,6 +4,9 @@ import * as moment from 'moment';
 import SCHEDULES from '~/assets/schedules.json';
 import * as chroma from 'chroma-js';
 
+// update this in SS19
+const splusStartDate = moment().startOf('year').year(2018);
```

Comment:
kann man da nicht das current year nehmen, also nicht "2018" etc.

Reply:
Ich gehe davon aus, dass in 2019 splus in den Wochen weiterläuft und erst im SS auf 2019-basierte Wochen wechselt. Der Plan geht bis KW65. Mit der Basis "aktuelles Jahr" wäre ein Benutzer, der die Anwendung nächstes Jahr aufruft, in (splus) 2018.